clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -240,7 +240,7 @@
                                         FETCH_MAX_WAIT_MS_DOC)
                                 .define(RECONNECT_BACKOFF_MS_CONFIG,
                                         Type.LONG,
-                                        50L,
+                                        500L,
```

Comment:
I'll revert this change since I didn't mean to commit it. 50ms seems a little low, but I'm not sure what is reasonable.